Client/game_sa/CWorldSA.cpp

```diff
@@ -354,6 +354,13 @@ bool CWorldSA::ProcessLineOfSight(const CVector* vecStart, const CVector* vecEnd
                     pBuildingResult->usLODModelID = 0;
 
                 pBuildingResult->pInterface = targetEntity;
+                if (targetEntity->Placeable.matrix)
```

Comment:
You could use `Placeable.m_transform.m_translate` if there is no `Placeable.matrix`